developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -3362,6 +3363,18 @@ int atoiW(PKMX_WCHAR p)
   return i;
 }
 
+bool isuiw(PKMX_WCHAR p) {
```

Comment:
I don't really know what this means -- can we use a longer name e.g. is_integer_wstring?

Reply:
See below.